components/apimgt/org.wso2.carbon.apimgt.rest.api.devops/src/main/java/org/wso2/carbon/apimgt/rest/api/devops/impl/LoggingApiServiceImpl.java

```diff
@@ -0,0 +1,92 @@
+package org.wso2.carbon.apimgt.rest.api.devops.impl;
```

Comment:
Let's add Licence header and remove wild card import. We need to do the same for other non gen files.